### PR TITLE
PCHR-1875: Fixed Action menu design break

### PR DESCRIPTION
**Pull request by mukeshcompucorp**
## Problem
Action menu was breaking due to fixed height/width and get extra menu item

ticket pchr-1875

## Solution
Set height/width auto and remove extra hardcoded menu item
Note: Dynamic menu items need fixing by backend developer

resolved-ticket pchr-1875

### CODE REVIEW

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 1]
```diff
@@ -20,13 +21,41 @@
     background-color: #FFF;
     padding: 10px 0 !important;
 
+    .crm-contact_activities-list{
+
+    }
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: This doesn't seem necessary

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 2]
```diff
@@ -20,13 +21,41 @@
...
     .crm-contact_actions-list {
       border-color: $crm-background !important;
+      width: 175px;
+      margin-left: 175px;
+      margin-right: 175px;
+      ul {
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: leave a new line

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 3]
```diff
@@ -20,13 +21,41 @@
...
+        .crm-action-new-case {
+          display: none;
+        }
+        .crm-action-new-relationship {
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: leave always new lines between selectors
```scs
.selector1 {

}

.selector2 {

}
```

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 4]
```diff
@@ -20,13 +21,41 @@
...
+          display: none;
+        }
+        .crm-action-new-note {
+          display: none;
+        }
+        .crm-action-tag-contact {
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: why don't you just group all the selectors? You are applying the same style for them
```scss
.selector1,
.selector2,
.selector3 {
  display: none;
}
```

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 5]
```diff
@@ -20,13 +21,41 @@
...
+          display: none;
+        }
+      }
+    }
+    .crm-contact_print-list{
+      width: 175px;
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Why do you need to specify a new width?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 6]
```diff
@@ -20,13 +21,41 @@
     background-color: #FFF;
     padding: 10px 0 !important;
 
+    .crm-contact_activities-list{
+
+    }
     .crm-contact_actions-list {
       border-color: $crm-background !important;
+      width: 175px;
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Why do you need to specify a new width?

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 2 above)
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: why the need for the `ul` selector? Can't you just do

```scss
.crm-contact_actions-list {
  
  .crm-action-new-case {

  }

  .crm-action-new-relationship {

  }
}
```

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss [hunk 7]
```diff
@@ -20,13 +21,41 @@
...
+          display: none;
+        }
+        .crm-action-tag-contact {
+          display: none;
+        }
+      }
+    }
+    .crm-contact_print-list{
+      width: 175px;
+      ul {
```
Comment (AkA84) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Same as before

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 6 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Text in action menu is longer and major links were satisfying this width over 160px so, it was better to increase it so, major cases gets covered.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 5 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Text in action menu is longer and major links were satisfying this width over 160px so, it was better to increase it so, major cases gets covered.

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 1 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Fixed

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 2 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Fixed

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 3 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Fixed

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 4 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Fixed

File: org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss (hunk 7 above)
Comment (mukeshcompucorp) [inline org.civicrm.bootstrapcivicrm/scss/civicrm/contact/_tooltip.scss]: Fixed

**Pull request merged**